Google.Api.Generator.Tests/ProtoTests/Lro/Testing.Lro/Testing.Lro.csproj

```diff
@@ -30,7 +30,7 @@
     -->
 
     <!-- These items should not require editing -->
-    <TargetFrameworks>netstandard1.5;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
This is the line that changes the targets. So why was it running against the old ones?